test/test_automl/test_estimators.py

```diff
@@ -704,3 +710,37 @@ def test_check_askl2_same_arguments_as_askl():
                            'metadata_directory']
     unexpected_args = set(extra_arguments) - set(expected_extra_args)
     assert len(unexpected_args) == 0, unexpected_args
+
+
+# which should return self actually return self.
+@pytest.mark.parametrize("task_type", ['classification',
+                                       'regression'])
+def test_fit_pipeline(dask_client, task_type):
+    X_train, y_train, X_test, y_test = putil.get_dataset(
+        'iris' if task_type == 'classification' else 'boston'
+    )
+    estimator = AutoSklearnClassifier if task_type == 'classification' else AutoSklearnRegressor
+    automl = estimator(
+        time_left_for_this_task=120,
+        # Time left for task plays no role
+        # only per run time limit
+        per_run_time_limit=30,
+        ensemble_size=0,
+        dask_client=dask_client,
+    )
+    config = automl.get_configuration_space(X_train, y_train).sample_configuration()
+
+    pipeline, run_info, run_value = automl.fit_pipeline(X=X_train, y=y_train, config=config)
+
+    assert isinstance(run_info.config, Configuration)
+    assert run_info.cutoff == 30
+    assert run_value.status == StatusType.SUCCESS
+    # We should produce a decent result
+    assert run_value.cost < 0.5
```

Comment:
you could also try the default configuration - then we would know the result.

Reply:
Thanks, I have done so